storage/local/chunk.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/prometheus/prometheus/storage/metric"
 )
 
+// The DefaultChunkEncoding can be changed via a flag.
```

Comment:
Side note: So this is just about newly created chunks, right?
Is there any remotely valid reason why one would choose the old encoding (i.e. virtually no compression)?